src/hydration/index.ts

```diff
@@ -1,11 +1,12 @@
-export { dehydrate, hydrate } from './hydration'
+export { dehydrate, hydrate } from '../core/hydration'
```

Comment:
Since the hydration package is built as a standalone package, this re-export will result in these functions being included in both core and hydration. I don't think this is a big problem since it's meant to be deprecated, there are no singletons there that could mess things up if you import things from both places, and the payload is small anyway.

This double payload _will_ affect everyone using the React bindings though since those are currently only included in /hydration. I wonder if this can be solved by changing this to `export { dehydrate, hydrate } from 'react-query'`? Not sure how re-exporting from a package configured as external works though, so would need to be tested.